client/consensus/babe/src/lib.rs

```diff
@@ -1875,13 +1875,14 @@ where
 	let finalized = client.info().finalized_number;
 	let revertible = blocks.min(best_number - finalized);
 
-	let number = best_number - revertible;
-	let hash = client
-		.block_hash_from_id(&BlockId::Number(number))?
-		.ok_or(ClientError::Backend(format!(
-			"Unexpected hash lookup failure for block number: {}",
-			number
-		)))?;
+	let revert_number = best_number - revertible;
+	let revert_hash =
```

Comment:
If you really want to make this more clear.

```suggestion
	let revert_up_to_number = best_number - revertible;
	let revert_up_to_hash =
```

But I'm also open for a better name, but `revert_*` is clearly the wrong naming.